clustering/src/main/java/io/hyperfoil/clustering/ControllerVerticle.java

```diff
@@ -48,6 +50,8 @@
 import io.vertx.core.spi.cluster.NodeListener;
 import io.vertx.ext.cluster.infinispan.InfinispanClusterManager;
 
+import org.infinispan.commons.api.BasicCacheContainer;
```

Comment:
Really?

Reply:
What do you mean by "Really?"
The import is needed for shutdown() and I don't believe I created shutdown().

Reply:
Sorry, I did not notice that you've just reordered imports (I thought this was added by accident).